### PR TITLE
Add component managers to README

### PR DESCRIPTION
**Pull request by Lunarmagpie**
### Summary

This PR adds the component managers to the README.

@FasterSpeeding I used the yuyo description from pypi. Please tell me to change it or push to my branch if you want something else.

### Checklist

- [x] I have run `nox` and all the pipelines have passed.
- [x] I have made unittests according to the code I have added/modified/deleted.

### CODE REVIEW

**Comment by Lunarmagpie** [comment]
Currently waiting on an update for the miru description from somebody lol

File: README.md [hunk 1]
```diff
@@ -255,6 +255,12 @@ Hikari does not include a command framework by default, so you will want to pick
 - [`tanjun`](https://github.com/FasterSpeeding/Tanjun) - a flexible command framework designed to extend Hikari.
 - [`crescent`](https://github.com/magpie-dev/hikari-crescent) - a command handler for Hikari that keeps your project neat and tidy.
 
+There are also third party libraries to help you manage components:
+
+- [`yuyo`](https://github.com/FasterSpeeding/Yuyo/) - a collection of utility functions and classes designed to enhances hikari.
```
Comment (HyperGH) [inline README.md:260]:
```suggestion
- [`yuyo`](https://github.com/FasterSpeeding/Yuyo/) - a collection of utility functions and classes designed to enhance hikari.
```

Comment (Lunarmagpie) [reply README.md]: fyi @FasterSpeeding this was copied directly from your project description on pypi so you may have to fix that there

File: README.md [hunk 2]
```diff
@@ -255,6 +255,12 @@ Hikari does not include a command framework by default, so you will want to pick
 - [`tanjun`](https://github.com/FasterSpeeding/Tanjun) - a flexible command framework designed to extend Hikari.
 - [`crescent`](https://github.com/magpie-dev/hikari-crescent) - a command handler for Hikari that keeps your project neat and tidy.
 
+There are also third party libraries to help you manage components:
+
+- [`yuyo`](https://github.com/FasterSpeeding/Yuyo/) - a collection of utility functions and classes designed to enhance hikari.
```
Comment (FasterSpeeding) [inline README.md:260]: Ngl including Yuyo here feels weird since its not a component library, it's just a general utility library.

Comment (Lunarmagpie) [reply README.md:260]: Ok, ill remove it

**Pull request closed without merging**